src/go/rpk/pkg/cli/cmd/iotune/iotune.go

```diff
@@ -39,17 +40,35 @@ func NewCommand(fs afero.Fs) *cobra.Command {
 			p := config.ParamsFromCommand(cmd)
 			cfg, err := p.Load(fs)
 			out.MaybeDie(err, "unable to load config: %v", err)
+
 			var evalDirectories []string
 			if len(directories) != 0 {
-				log.Infof("Overriding evaluation directories with '%v'",
+				fmt.Printf("Overriding evaluation directories with: %q\n",
 					directories)
 				evalDirectories = directories
 			} else {
 				evalDirectories = []string{cfg.Redpanda.Directory}
 			}
 
-			err = execIoTune(fs, evalDirectories, outputFile, duration, timeout)
-			out.MaybeDie(err, "error during IoTune execution: %v", err)
+			if exists, _ := afero.Exists(fs, outputFile); exists && !noConfirm {
+				confirmed, err := out.Confirm("IO config file detected, do you want to continue? doing this will overwrite previous results.")
+				out.MaybeDie(err, "unable to confirm execution: %v", err)
+				if !confirmed {
+					out.Exit("IO tune canceled.")
+				}
+			}
+			tuner := tuners.NewIoTuneTuner(
+				fs,
+				evalDirectories,
+				outputFile,
+				duration,
+				timeout,
+			)
+			fmt.Println("Starting iotune...")
+			result := tuner.Tune()
+			out.MaybeDie(result.Error(), "error during IoTune execution: %v", result.Error())
```

Comment:
Please make the spelling of `iotune` consistent. There's `IO tune`, `iotune` and `IoTune`. Since the binary is `iotune`, I think we should stick with that.